core/src/main/java/io/grpc/CompressorRegistry.java

```diff
@@ -49,10 +49,16 @@
       new Codec.Gzip(),
       Codec.Identity.NONE);
 
+  /**
+   * Returns the default instance which supports gzip compression.
```

Comment:
I wonder if we really want to mention "gzip" here. Are we allowed to change the default compression in the future? @ejona86